workspace/index.js

```diff
@@ -119,11 +119,6 @@ module.run(function ($log, $q, $http, ImpacLinking, ImpacAssets, ImpacRoutes, Im
     }
   });
 
-  // Link Impac! Assets
```

Comment:
ok, I missed this one: 
workspace/assets/impac-logo.png → src/images/impac-title-logo.png
:+1: